package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "ember-cli-version-checker": "^1.2.0",
     "execa": "^0.6.0",
-    "fs-extra": "^2.0.0",
+    "fs-extra": "^3.0.0",
```

Comment:
That's a different reason to upgrade though. If we benefit from the new version that makes sense. I'm opposed to only upgrading because a new version is **available**.